doc/source/analyzing/generating_processed_data.rst

```diff
@@ -9,6 +9,42 @@ data by hand and construct plots which can then be combined with other plots,
 modified in some way, or even (gasp) created and modified in some other tool or
 program.
 
+.. _exporting-container-data:
+
+Exporting Container Data
+------------------------
+
+Fields from data containers such as regions, spheres, cylinders, etc. can be exported
+tabular format using either a :class:`~pandas.DataFrame` or an :class:`~astropy.table.QTable`. 
+
+To export to a :class:`~pandas.DataFrame`, use 
+:meth:`~yt.data_objects.data_containers.YTDataContainer.to_dataframe`:
+
+.. code-block:: python
+
+    sp = ds.sphere("c", (0.2, "unitary"))
+    # Adds all of the fields to the DataFrame which have been loaded into memory
+    # already
+    df = sp.to_dataframe()
+    # Only adds the density and temperature fields
+    df2 = sp.to_dataframe(fields=["density","temperature"])
+
+To export to a :class:`~astropy.table.QTable`, use 
+:meth:`~yt.data_objects.data_containers.YTDataContainer.to_astropy_table`:
+
+.. code-block:: python
+
+    sp = ds.sphere("c", (0.2, "unitary"))
+    # Adds all of the fields to the Table which have been loaded into memory
+    # already
+    df = sp.to_astropy_table()
```

Comment:
extremely minor nit here, but I feel like calling this variable `df` implies it is a DataFrame, when that's not what you're creating here.

Reply:
I think this was a copied line which wasn't fixed after the fact. Thanks for noticing!